woocommerce/woocommerce-gateway-stripe
# Feature: Add UPE availability inbox notice

Fixes #1692

### Description
Adding an inbox notice for the merchants informing them about the availability of the new payment experience in Stripe.

![inbox](https://user-images.githubusercontent.com/33387139/131287918-39b68575-89d4-477d-a465-79b9eb36ef3a.png)

### Testing instructions
- To make the notice visible in the inbox 
-> Connect Stripe in your store to make the notice available
-> Enable UPE preview flag (returning `true` from [this function](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/24277919aa7b9029c39ecd997dd133990e7f65c7/includes/class-wc-stripe-feature-flags.php#L15) should do the work)
-> UPE should be disabled (returning `false` from [this function](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/24277919aa7b9029c39ecd997dd133990e7f65c7/includes/class-wc-stripe-feature-flags.php#L24) should do the work)
- To check the conditions when notice should not be visible 
-> delete the already created notice from DB (`wp_wc_admin_notes` and `wp_wc_admin_note_actions` table) 
-> Make sure the conditions mentioned above are unsupported
-> Refresh the page

### QA checklist

- [ ] When Stripe is connected, UPE is disabled and UPE preview flag is enabled a new inbox notice titled `Boost your sales with the new payment experience in Stripe` appears.
- [ ] The notice does not appear if the conditions are unsupported.
